src/modules/flow/persistence/persistence.c

```diff
@@ -47,18 +47,10 @@ struct storage_fn {
 };
 
 struct persist_data {
+    const struct storage_fn *storage;
     void *value_ptr;
-
     char *name;
     char *fs_dir_path;
-
-    struct sol_flow_packet *(*packet_new_fn)(const struct persist_data *data);
```

Comment:
Woot, ``packet_new_fn`` was not being called? The rest of the patch seems fine :)